tgarchive/example/template.html

```diff
@@ -39,163 +40,171 @@
 			<header class="header">
 				<div class="logo">
 					<a href="{{ config.site_url }}"><img src="static/logo.svg" alt="" /></a>
-					<p class="desc">
+					<div class="desc">
 						<a href="{{ config.telegram_url.format(id=config.group) }}" rel="noreferer nopener nofollow">@{{ config.group }}</a> /
 						{{ config.site_description.format(group=config.group, date=config.date) }}
-					</p>
+					</div>
 				</div>
 			</header>
-			<ul class="timeline index">
-		        {% for year, months in timeline.items() | reverse %}
-		        <li class="">
-		        	<h3 class="year"><a href="{{ months[0].slug }}.html">{{ year }}</a></h3>
-		        	<ul class="months">
-		        		{% for m in months | reverse %}
-		        			<li class="{% if m.slug == month.slug %}selected{% endif %}">
-		        				<a href="{{ m.slug }}.html">
-		        					{{ m.label }}
-		        					<span class="count">({{ m.count }})
-		        				</span></a>
-		        			</li>
-		        		{% endfor %}
-		        	</ul>
-		        </li>
-		        {% endfor %}
+			<ul class="timeline index">{%
```

Comment:
This is very difficult to read. See above comment about indentation being important in the raw template. It has no importance in the generated HTML.